app-editors/bluefish/bluefish-2.2.12-r2.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
```

Comment:
not quite sure if I can just increase this to python 11. thoughts?

Reply:
You have to do a test build w/ USE=python and make sure it works, it's ok to skip it for now given we're trying to do so many clang bugs